src/components/empty/EmptyTable.tsx

```diff
@@ -15,9 +16,11 @@ interface IEmptyTable {
 }
 
 export const EmptyTable: FC<IEmptyTable> = ({ title, description }) => {
+  const ref = useRef<HTMLDivElement>();
+  const { height } = useDimensions(ref);
   return (
-    <SEmptyTable>
-      {[...Array(5)].map((_, index) => (
+    <SEmptyTable ref={ref}>
```

Comment:
@mahmoudtahadev  i do not see the need to get the height during the runtime and fill the whole table - rather keep the fixed amount of 5 skeleton rows -> then you can adjust the style to the height to the end of this 5 rows since you always know that the rows will be 5* rowsize and then you can let the gradient end where it needs to end